src/bin/jcli_app/block/genesis/yaml.rs

```diff
@@ -308,10 +294,13 @@ impl BlockchainConfiguration {
                 ConfigParam::EpochStabilityDepth(param) => epoch_stability_depth
                     .replace(*param)
                     .map(|_| "EpochStabilityDepth"),
-                ConfigParam::ConsensusLeaderId(param) => {
+                ConfigParam::AddBftLeader(param) => {
                     consensus_leader_ids.push(SerdeLeaderId(param.clone()));
                     None
                 }
+                ConfigParam::RemoveBftLeader(_) => {
```

Comment:
Does this mean that config is now not declarative, but procedural?

Reply:
BFT nodes aren't immutable.

Reply:
Yeah that's right. Also, `ConfigParams` is now best seen as a diff to the "current" configuration (since that's what an update proposal describes).